docs: warn users not to use cpu training with a custom scheduler (#2703)
determined-ai/determined: docs/how-to/installation/kubernetes.txt

   defaultScheduler: preemption

> If you are using cpu training with Kubernetes, avoid setting the

Reviewer: ```suggestion
If you are using CPU-only training with Kubernetes, avoid setting the
```

> ``defaultScheduler`` field. Cpu training has not yet been validated for

Reviewer: ```suggestion
``defaultScheduler`` field. CPU training is not supported for these scheduling options yet.
```

Reviewer: I don't see these changes applied after the last force-push?..
